src/lib/browse/Filters.svelte

```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+  import { fundingProgrammesForColouringAndFiltering } from "lib/browse/data";
```

Comment:
You can do `./data`. We've been doing the "absolute paths" when it's shared across the codebase, but this is "local" to this directory and unlikely to move